format-blob/pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Copyright © 2018-2019 Cask Data, Inc.
+  ~ Copyright © 2022 Cask Data, Inc.
```

Comment:
existing copyright years should not be replaced. Instead, the upper bound should be updated. In this case, 2018-2022.

Reply:
Done